closed/src/java.base/share/classes/com/sun/crypto/provider/NativeGaloisCounterMode.java

```diff
@@ -97,6 +99,16 @@
         nativeCrypto = NativeCrypto.getNativeCrypto();
     }
 
+    // check if the specified length (in bytes) is a valid keysize for AES
+    static final boolean isKeySizeValid(int len) {
```

Comment:
Does it make sense to reuse the implementation from `NativeCipherBlockChaining`?
If so, you might also refactor this so instead of a boolean function, we have one that throws `InvalidKeyException` if appropriate.

Reply:
just realized I can use `isKeySizeValid()` from AESCrypt.java directly, so I changed it to that.